### ISSUE TITLE
Etching fails miserably if wallet goes offline during etching

### ISSUE THREAD

**Issue by embiimob** [HugPuddle/Apertus#20]
a re-write of the etching process to make it transaction based, pause-able and recoverable

**Comment by baffo32** [comment]
Etching should use OP_RETURN too

**Comment by embiimob** [comment]
Agreed , 0.4.0 will support DataCoin florincoin data storage specifications  as well as standard op_return of various sizes

**Comment by embiimob** [comment]
it appears newer bitcoin wallets goes offline after 20 etches in rapid succession  I have changed the confirmation wait to trigger after 15 transactions.  this should limit failed etchings... will release a quick fix release later tonight.  This does not solve the problem above...but appears to be the cause of the most recent batch of failed etchings.    Cat and mouse :dancer:
